### PR TITLE
upgrade elastic chart; fix zoom in bug

### PR DESCRIPTION
**Pull request by ylwu-amzn**
*Issue #, if available:*

*Description of changes:*
Upgrade elastic chart;
Fix AD result chart zoom in bug

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: public/pages/AnomalyCharts/containers/__tests__/AnomaliesChart.test.tsx [hunk 1]
```diff
@@ -74,6 +74,6 @@ describe('<AnomaliesChart /> spec', () => {
   test('renders the component', () => {
     console.error = jest.fn();
     const { getByText } = renderDataFilter();
-    expect(getByText('anomaly grade')).not.toBeNull();
+    expect(getByText('Sample anomaly grade')).not.toBeNull();
```
Comment (ohltyler) [inline public/pages/AnomalyCharts/containers/__tests__/AnomaliesChart.test.tsx:77]: Where is 'Sample anomaly grade' rendered on anomalies chart page?

**Comment by ylwu-amzn** [comment]
```
Test Suites: 1 skipped, 45 passed, 45 of 46 total
Tests:       4 skipped, 198 passed, 202 total
Snapshots:   35 passed, 35 total
Time:        110.497s, estimated 116s
Ran all test suites.
✨  Done in 113.45s.
```

```
  AD Dashboard
    ✓ Empty dashboard - no detector index (27653ms)
    ✓ Empty dashboard - empty detector index (15526ms)
    ✓ AD dashboard - single stopped detector (20423ms)
    ✓ AD dashboard - redirect to create detector (13189ms)
    ✓ Filter by detector (18761ms)
    ✓ Filter by detector state (14320ms)


  6 passing (2m)
```

File: public/pages/AnomalyCharts/containers/__tests__/AnomaliesChart.test.tsx (hunk 1 above)
Comment (ylwu-amzn) [reply public/pages/AnomalyCharts/containers/__tests__/AnomaliesChart.test.tsx:77]: https://github.com/opendistro-for-elasticsearch/anomaly-detection-kibana-plugin/blob/master/public/pages/AnomalyCharts/containers/AnomaliesChart.tsx#L280

File: package.json [hunk 2]
```diff
@@ -76,7 +76,7 @@
     "typescript": "3.0.3"
   },
   "dependencies": {
-    "@elastic/charts": "^18.2.2",
+    "@elastic/charts": "^19.2.0",
```
Comment (yizheliu-amazon) [inline package.json:79]: can you try removing this dependency? After removing it, we should automatically use [elastic chart of Kibana](https://github.com/elastic/kibana/blob/master/package.json#L124) instead

Comment (ylwu-amzn) [reply package.json:79]: Good point. Sorry that I missed this reply. Will remove it in next PR.

**Pull request merged**